frontend/src/components/explorer/gemBrowser/Metabolite.vue

```diff
@@ -70,6 +70,12 @@
               </div>
               <ExtIdTable :type="type" :external-dbs="metabolite.externalDbs"></ExtIdTable>
             </div>
+            <div v-if="metabolite && metabolite.externalDbs && metabolite.externalDbs.ChEBI"
```

Comment:
This is a nice addition but I think we could find a better placement for it. Suggestions are:

1. Inline with the table, perhaps between Formula and Charge.
2. In the sidebar at the top.

Reply:
In both cases, the image looked to small to be meaningful - it seemed more like a logo than an image that conveys useful information.

Reply:
Sounds reasonable. If we are keeping the image where it is I think we should consider making two changes.

1. Add a faint border/frame and a caption for the image. There is a lot of vertical whitespace right now.
2. If there is no chebi image, then use the original layout (table width). Since the table is always the same size now, the sidebar is not aligned to the right of the page.